applications/openshift/kubelet/kubelet_enable_streaming_connections/rule.yml

```diff
@@ -31,7 +36,7 @@ ocil_clause: 'the streaming connection timeouts are not disabled'
 
 ocil: |-
     Run the following command on the kubelet node(s):
-    <pre>$ sudo grep streamingConnectionIdleTimeout {{{ kubeletconf_path }}}</pre>
+    <pre>$ oc get --raw /api/v1/nodes/${NODE_NAME}/proxy/configz | jq '.kubeletconfig|.kind="KubeletConfiguration"|.apiVersion="kubelet.config.k8s.io/v1beta1"' | grep streamingConnectionIdleTimeout</pre>
```

Comment:
This is maybe a nit, but we're already getting complains from support that our OCILs are nto actionable for the customers, so let's fix them while we're touching the rules..we should fix the OCILs so that the commands iterate over the nodes, for example this:
```
 for NODE_NAME in $(oc get nodes -ojsonpath='{.items[*].metadata.name}'); do oc get --raw /api/v1/nodes/$NODE_NAME/proxy/configz | jq '.kubeletconfig|.kind="KubeletConfiguration"|.apiVersion="kubelet.config.k8s.io/v1beta1"' | grep streamingConnectionIdleTimeout; done
```

Reply:
I think that's a good idea, we should fix the OCILs